Backend/src/main/java/ca/mcgill/cooperator/controller/CourseOfferingController.java

```diff
@@ -38,10 +43,10 @@ public CourseOfferingDto getCourseOfferingById(@PathVariable int id) {
     }
 
     @GetMapping("/course/{id}")
-    public List<CourseOfferingDto> getCourseOfferingsByCourse(@PathVariable int id) {
+    public Set<CourseOfferingDto> getCourseOfferingsByCourse(@PathVariable int id) {
```

Comment:
Why did you change this to a set?

Reply:
because i need to do the set intersections and you cant use lists for that

Reply:
Ah right okay makes sense